cmd/policy/build.go

```diff
@@ -23,9 +23,15 @@ type BuildCmd struct {
 	ExcludeVerifyFiles []string `name:"exclude-files-verify" help:"Set file names to exclude during bundle verification."`
 	SigningKey         string   `name:"signing-key" help:"Set the secret (HMAC) or path of the PEM file containing the private key (RSA and ECDSA)."`
 	ClaimsFile         string   `name:"claims-file" help:"Set path of JSON file containing optional claims (see: https://openpolicyagent.org/docs/latest/management/#signature-format)."`
+	RegoVersion        string   `name:"rego-version" enum:"default,rego.v1" default:"default" help:"specify rego version flag (enum: default or rego.v1)"`
```

Comment:
Please make the help message start with a capital letter, like the rest of the messages.

```
policy build --help
Usage: policy build <path> ...

Build policies.

Arguments:
  <path> ...    Path to the policy sources.

Flags:
  -h, --help                                                Show context-sensitive help.
  -c, --config="/Users/gertd/.config/policy/config.yaml"    Path to the policy CLI config file.
      --debug                                               Enable debug mode.
  -v, --verbosity=INT                                       Use to increase output verbosity.
  -k, --insecure                                            Do not verify TLS connections.

  -t, --tag=STRING                                          Name and optionally a tag in the 'name:tag' format, if not provided it will be 'default:latest'
  -a, --annotations=KEY=VALUE;...                           Annotations to apply to the policy.
      --build-config-file=STRING                            Set path of configuration file.
      --target="rego"                                       Set the output bundle target type.
  -O, --optimize=0                                          Set optimization level.
  -e, --entrypoint=ENTRYPOINT,...                           Set slash separated entrypoint path.
  -r, --revision=STRING                                     Set output bundle revision.
      --ignore=IGNORE,...                                   Set file and directory names to ignore during loading (e.g., '.*' excludes hidden files).
      --capabilities=STRING                                 Set capabilities.json file path.
      --verification-key=STRING                             Set the secret (HMAC) or path of the PEM file containing the public key (RSA and ECDSA).
      --verification-key-id="default"                       Name assigned to the verification key used for bundle verification.
      --signing-alg="RS256"                                 Name of the signing algorithm.
      --scope=STRING                                        Scope to use for bundle signature verification.
      --exclude-files-verify=EXCLUDE-FILES-VERIFY,...       Set file names to exclude during bundle verification.
      --signing-key=STRING                                  Set the secret (HMAC) or path of the PEM file containing the private key (RSA and ECDSA).
      --claims-file=STRING                                  Set path of JSON file containing optional claims (see: https://openpolicyagent.org/docs/latest/management/#signature-format).
      --rego-version="default"                              specify rego version flag (enum: default or rego.v1)
```

Suggest to update the message using similar terminology as OPA in https://www.openpolicyagent.org/docs/latest/policy-language/#the-regov1-import

as this flag effectively enables `import rego.v1` would be useful if the message relays this information.

The docs reference the OPA https://www.openpolicyagent.org/docs/latest/policy-language/#the-regov1-import link and emphasize that this is about current or future compatibility.